geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/services/SynchronizedStripedCoordinator.java

```diff
@@ -51,6 +52,17 @@ public SynchronizedStripedCoordinator(int concurrencyLevel) {
     }
   }
 
+  @Override
+  public <T> T execute(List<Object> stripeIds, int index, Callable<T> callable) {
```

Comment:
Initially it needed this otherwise these signatures would be ambiguous:
```
  public <T> T execute(Object stripeId, Callable<T> callable)
  public <T> T execute(List<Object> stripeIds, Callable<T> callable)
```
I've applied Donal's suggestion and made the locked object explicitly be a `RedisKey`.